Thyra Collection API Testing.postman_collection.json

```diff
@@ -0,0 +1,56 @@
+{
+	"info": {
+		"_postman_id": "87c5b13d-f24d-44e4-95b5-0e11a352f384",
+		"name": "Thyra Collection API Testing",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "24086813"
+	},
+	"item": [
+		{
+			"name": "/cmd/deploySC",
+			"protocolProfileBehavior": {
+				"disabledSystemHeaders": {}
+			},
+			"request": {
+				"method": "PUT",
+				"header": [
+					{
+						"key": "Content-Type",
+						"value": "application/json",
+						"type": "text",
+						"disabled": true
+					}
+				],
+				"body": {
+					"mode": "formdata",
+					"formdata": [
+						{
+							"key": "Wasmfile",
+							"type": "file",
```

Comment:
Not sure this is cross plateforme. Can you try to find a way to solve that and make it work for the CI ?

Reply:
To be addressed in #504